index.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>FIFA World Cup 2022 - home</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap" rel="stylesheet">
+</head>
+<body>
+    <!-- Header -->
+    <header>
+        <nav class="nav-bar">
```

Comment:
Great job @hafiz1379 🤝

- [ ] [Optional] Kindly ensure your navbars are well aligned so that your design can match the design provided.
- Your design
![Your-navbar](https://user-images.githubusercontent.com/63917725/234538790-97fbd04d-6a21-463f-9547-9c042540e0cf.PNG)
- Given design
![Given-navbar](https://user-images.githubusercontent.com/63917725/234538858-7f1931dd-b158-4f39-89e3-c6752773a1d8.PNG)